bundle-workflow/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Copyright OpenSearch Contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+DIR="$(dirname "$0")"
+
+if [ -z "$1" ]; then (echo "syntax: run.sh [workflow.py]"; exit -1); fi
+command -v python3 >/dev/null 2>&1 || (echo "missing python3, try 'sudo apt install python3'"; exit -1)
+command -v pip >/dev/null 2>&1 || (echo "missing pip, try 'sudo apt install python3-pip'"; exit -1)
+command -v pipenv >/dev/null 2>&1 || (echo "missing pipenv, try 'sudo apt install pipenv'"; exit -1)
```

Comment:
On line 15 we install pipenv, so we shouldn't fail on line 13 if it isn't already installed.

Also, `apt` is not the recommended way to install python dependencies, including pip. See https://pip.pypa.io/en/stable/installation/ for better options.

Reply:
> On line 15 we install pipenv, so we shouldn't fail on line 13 if it isn't already installed.

That's incorrect, on line 15 we install all the dependencies that are listed in Pipfile.lock, not pipenv.
 
> Also, `apt` is not the recommended way to install python dependencies, including pip. See https://pip.pypa.io/en/stable/installation/ for better options.

That's why we don't. I think you might want to read up on [pipenv](https://pipenv.pypa.io/en/latest/). We install python, pip and pipenv, which are all globally installed tools. Then we install all the other dependencies by running `pipenv install` from `Pipfile.lock`.

Reply:
We say "try 'sudo apt install pipenv'"...we shouldn't give that advice. We should direct people to the pipenv website for installation instructions. Same for line 12: apt is not the best way to install pip.

Reply:
Removed advice, updated to just say "missing pipenv" and such.